makefile

```diff
@@ -0,0 +1,9 @@
+FETCH_SCRIPT_DIR = scripts/fetch_data
+PROCESS_SCRIPT_DIR = scripts/process_data
+
+all: fetch_data process_data
+
+fetch_data: Rscript $(FETCH_SCRIPT_DIR)/fetch_data.R
+
+process_data: 
+	Rscript $(PROCESS_SCRIPT_DIR)/fitbitactivitylogs.R
```

Comment:
what about the rest of the files under $(PROCESS_SCRIPT_DIR)/? fitbitdailydata, fitbitsleeplogs etc.,?

Reply:
The makefile was a prototype and not meant to be included for now. Thanks for catching that, I am going to remove the file from this branch.